openmc/mgxs/mgxs.py

```diff
@@ -1625,14 +1773,12 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
 
         # If the user requested a specific set of nuclides
         elif self.by_nuclide and nuclides != 'all':
-            query_nuclides = nuclides
```

Comment:
Ah yes, it was used in the old way to override the enrgy group bounds, and now i avoid that.